src/dbnode/storage/bootstrap/bootstrapper/peers/source_index_test.go

```diff
@@ -468,7 +468,7 @@ func TestBootstrapIndex(t *testing.T) {
 //			_ result.Options,
 //		) (result.ShardResult, error) {
 //			goodID := ident.StringID("foo")
-//			goodResult := result.NewShardResult(0, opts.ResultOptions())
+//			goodResult := result.NewShardResult(opts.ResultOptions())
```

Comment:
nit: should this commented section just be deleted?

Reply:
Sure thing.